google/auth/exceptions.py

```diff
@@ -57,3 +57,12 @@ def __init__(self, message=None):
         super(ReauthFailError, self).__init__(
             "Reauthentication failed. {0}".format(message)
         )
+
+
+class ReauthSamlChallengeFailError(ReauthFailError):
+    """An exception for SAML reauth challenge failures."""
+
+    def __init__(self, message=None):
+        super(ReauthSamlChallengeFailError, self).__init__(
+            "Reauthentication failed. {0}".format(message)
```

Comment:
"Reauthentication failed" appears in the base class too, so we will see this message twice? I seems we don't have to define a __init__ for this class because we can use the __init__ of the base class.

Reply:
removed